apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Snippets/BigtableClientSnippets.cs

```diff
@@ -0,0 +1,367 @@
+                        "metrics",
+                        "clicks",
+                        new BigtableVersionRange(null, DateTime.UtcNow.AddDays(-1)))),
+                // Delete all data from row 'user23456'
+                Mutations.CreateEntry("user23456", Mutations.DeleteFromRow()));
+
+            // Read streaming responses from server until complete
+            IAsyncEnumerator<MutateRowsResponse> responseStream = streamingResponse.ResponseStream;
+            while (await responseStream.MoveNext(cancellationToken))
+            {
+                MutateRowsResponse response = responseStream.Current;
+                foreach (MutateRowsResponse.Types.Entry entry in response.Entries)
+                {
+                    switch (entry.Index)
+                    {
+                        case 0:
+                            Console.WriteLine($"The mutations to row 'user12345' finished with status code {entry.Status.Code}");
+                            break;
+                        case 1:
+                            Console.WriteLine($"The mutations to row 'user23456' finished with status code {entry.Status.Code}");
+                            break;
+                        default:
+                            throw new ArgumentOutOfRangeException();
+                    }
+                }
+            }
+            // End snippet
+
+            await BigtableAssert.HasNoValuesAsync(
+                client,
+                new TableName(projectId, instanceId, tableId),
+                "user23456");
+        }
+
+        [Fact]
+        public async Task ReadModifyWriteRowAsync()
+        {
+            var client = _fixture.TableClient;
+            var projectId = _fixture.TableName.ProjectId;
+            var instanceId = _fixture.TableName.InstanceId;
+            var tableId = _fixture.TableName.TableId;
+
+            await client.MutateRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                Mutations.SetCell("Score", "Level 1", 12));
+
+            // Snippet: ReadModifyWriteRowAsync(TableName,BigtableByteString,*)
+            // Increase the value of the cell with the "Level 1" qualifier in the
+            // "Score" column family by 55.
+            ReadModifyWriteRowResponse response = await client.ReadModifyWriteRowAsync(
+                new TableName(projectId, instanceId, tableId),
+                "user12345",
+                ReadModifyWriteRules.Increment("Score", "Level 1", 55));
+
+            Cell modifiedCell = response.Row.Families[0].Columns[0].Cells[0];
+
+            // Implicitly convert the ByteString Value into a BigtableByteString,
+            // which supports additional conversions specific to Bigtable.
+            BigtableByteString value = modifiedCell.Value;
+
+            // Convert the BigtableByteString to a long to obtain the updated value.
+            long newScore = (long)value;
```

Comment:
Maybe print this out? It's odd to have a snippet which ends with a conversion.
(I'd also personally have a space: `(long) value` - but I'm not overly bothered on that front.)

Reply:
Done